utils/Constants.sol

```diff
@@ -304,6 +299,17 @@ contract Constants {
         }
     }
 
+    function getCauldron(string calldata chain, string memory name, uint8 version) public view returns (CauldronInfo memory) {
+        CauldronInfo[] memory cauldronInfos = cauldronsPerChain[chain];
+
+        for (uint256 i = 0; i < cauldronInfos.length; i++) {
+            CauldronInfo memory info = cauldronInfos[i];
+            if (info.version == version && keccak256(abi.encodePacked(info.name)) == keccak256(abi.encodePacked(name))) return info;
+        }
+
+        revert("cauldron not found");
```

Comment:
you can do `revert(string.concat("cauldron ", name, " not found"))` instead

Reply:
Nice. I like this.